appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/admin/GetMetricsConfigurationCommand.java

```diff
@@ -0,0 +1,122 @@
+ *
+ * Contributor(s):
+ * If you wish your version of this file to be governed by only the CDDL or
+ * only the GPL Version 2, indicate your decision by adding "[Contributor]
+ * elects to include this software in this distribution under the [CDDL or GPL
+ * Version 2] license."  If you don't indicate a single choice of license, a
+ * recipient has the option to distribute your version of this file under
+ * either the CDDL, the GPL Version 2 or to extend the choice of license to
+ * its licensees as provided above.  However, if you add GPL Version 2 code
+ * and therefore, elected the GPL Version 2 license, then the option applies
+ * only if the new code is made subject to such option by the copyright
+ * holder.
+ */
+package fish.payara.microprofile.metrics.admin;
+
+import com.sun.enterprise.config.serverbeans.Config;
+import com.sun.enterprise.util.ColumnFormatter;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import javax.inject.Inject;
+import org.glassfish.api.ActionReport;
+import org.glassfish.api.Param;
+import org.glassfish.api.admin.AdminCommand;
+import org.glassfish.api.admin.AdminCommandContext;
+import org.glassfish.api.admin.ExecuteOn;
+import org.glassfish.api.admin.RestEndpoint;
+import static org.glassfish.api.admin.RestEndpoint.OpType.GET;
+import org.glassfish.api.admin.RestEndpoints;
+import org.glassfish.api.admin.RuntimeType;
+import static org.glassfish.config.support.CommandTarget.CLUSTER;
+import static org.glassfish.config.support.CommandTarget.CLUSTERED_INSTANCE;
+import static org.glassfish.config.support.CommandTarget.CONFIG;
+import static org.glassfish.config.support.CommandTarget.DAS;
+import static org.glassfish.config.support.CommandTarget.STANDALONE_INSTANCE;
+import org.glassfish.config.support.TargetType;
+import org.glassfish.hk2.api.PerLookup;
+import org.glassfish.internal.api.Target;
+import org.jvnet.hk2.annotations.Service;
+
+/**
+ * AsAdmin command to get metrics configuration
+ * 
+ * @author Gaurav Gupta
+ */
+@Service(name = "get-metrics-configuration")
+@PerLookup
+@ExecuteOn({RuntimeType.DAS})
+@TargetType({DAS, STANDALONE_INSTANCE, CLUSTER, CLUSTERED_INSTANCE, CONFIG})
+@RestEndpoints({
+    @RestEndpoint(configBean = MetricsServiceConfiguration.class,
+            opType = GET,
+            path = "get-metrics-configuration",
+            description = "Gets the Metrics Configuration")
+})
+public class GetMetricsConfigurationCommand implements AdminCommand {
+
+    private final String OUTPUT_HEADERS[] = {"Enabled", "Secure"};
+    
+    @Inject
+    private Target targetUtil;
+    
+    @Param(optional = true, defaultValue = "server-config")
+    private String target;
+    
+    @Override
+    public void execute(AdminCommandContext acc) {
+        Config targetConfig = targetUtil.getConfig(target);
+        
+        if (targetConfig == null) {
+            acc.getActionReport().setMessage("No such config name: " + targetUtil);
```

Comment:
Better to prevent uncommon abbreviations like "acc". Could be "commandContext" or something.